.eslintrc

```diff
@@ -42,7 +42,13 @@
     "import/no-unresolved": "off",
     "import/extensions": "off",
     "react/prop-types": "off",
-    "prettier/prettier": "error"
+    "prettier/prettier": [
+      "error",
+      {
```

Comment:
This now duplicates the options that are also defined in `.prettierrc`. I wonder if there's a way to make sure it's reading that file instead here?